src/common/components/ui/formik/utils.ts

```diff
@@ -53,32 +46,57 @@ export const parseStringLabels = (
   return labels;
 };
 
-// Input: ['foo=bar', 'key=value', 'foo=blee']
-// Result: ['key=value', 'foo=blee']
-export const uniqueLabels = (labelPairs: string[]): string[] =>
-  labelsToArray(parseStringLabels(labelPairs));
+type innerError = { path: string; message: string };
+type innerErrors = { inner: innerError[] };
+type fieldErrors = { [key: string]: string[] };
```

Comment:
types should start with capital letter

Reply:
Fixed, thanks.
Will check if we can enforce it using `https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/naming-convention.md`